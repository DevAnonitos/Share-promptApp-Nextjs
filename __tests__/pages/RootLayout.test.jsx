import React from "react";
import { render, screen } from "@testing-library/react";
import RootLayout from "@/app/layout";

describe("MainLayout", () => {
  it("should render RootLayout component", () => {
    expect(RootLayout).toBeDefined()
    expect(RootLayout).toMatchSnapshot()
  });
});