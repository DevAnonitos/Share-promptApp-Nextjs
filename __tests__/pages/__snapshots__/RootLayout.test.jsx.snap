// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout should render RootLayout component 1`] = `[Function]`;
